src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -90,6 +90,8 @@ protected function match_existing_post( array $record ) {
 	protected function update_post( $post_id, array $record ) {
 		$update_authority_setting = tribe( 'events-aggregator.settings' )->default_update_authority( 'csv' );
 
+		$this->watch_term_creation();
```

Comment:
Should there be an unwatch (remove_action) call after everything is run in this method?

Reply:
Good idea, added.